redis/src/cluster_slotmap.rs

```diff
@@ -171,18 +206,31 @@ impl Display for SlotMap {
                 "({}-{}): primary: {}, replicas: {:?}",
                 slot_map_value.start,
                 end,
-                slot_map_value.addrs.primary,
-                slot_map_value.addrs.replicas
+                slot_map_value.addrs.read().unwrap().primary,
```

Comment:
better capture the lock once and use it rather of locking it twice.
Although this is thread-safe, it is not-atomic. i.e we might print incorrect values

Reply:
done